ddp-workspace/projects/ddp-pancan/src/app/components/navigation/navigation.component.ts

```diff
@@ -0,0 +1,32 @@
+import { Component } from '@angular/core';
+
+@Component({
+    selector: 'app-navigation',
+    template: `
+    <nav>
+        <ul class="navigation">
+            <li class="navigation__item">
+                <a class="navigation__link" routerLink="about-us" routerLinkActive="header-link_active" translate>
```

Comment:
Can you please use constants from `app-routes.ts` ?

Reply:
thanks for noticing! fixed